pkg/envtest/server.go

```diff
@@ -113,6 +116,10 @@ type Environment struct {
 	// values are merged.
 	CRDDirectoryPaths []string
 
+	// BinaryDirectoryPath is the path where the binaries required for the envtest are
+	// locate in the environment
+	BinaryDirectoryPath string
```

Comment:
Nit: if envtest requires an absolute path, this is the variable name to use
```suggestion
	AbsBinaryDirectory string
```

Reply:
I don't think anything in this requires it be an absolute path? Though definitely would be playing with fire to do otherwise :)

Reply:
Just checked, it isn't. It definitely should be because theoretically you could run the test from anywhere in the project, which would probably mess with how envtest processes are spawned.

Reply:
Changed as suggested.